### DISCUSSION TITLE
Its keeps disconnecting and reconnecting every 1-2 minutes

### DISCUSSION THREAD

**Discussion by Celestin6669** [rustdesk/rustdesk#5811] [General]
I live in Kazakhstan and both on my work and home stations are keep disconnecting therefore i cant connect to anyone and no1 cant connect to me. Doesnt matter on which provider or wifi or 4-5g phone network.

**Comment by dinger1986** [comment]
are you self hosting?

**Reply by Celestin6669** [reply]
no, i  havent tried it.

**Comment by Celestin6669** [comment]
from wireshark it seems like it keeps calling for udp port, but idk why it keeps dropping connection
245	129.998684	192.168.0.101	155.138.247.159	UDP	56	54657 → 21116 Len=14
246	130.788432	192.168.0.1	192.168.0.255	UDP	370	21015 → 20002 Len=328
250	132.002719	192.168.0.101	155.138.247.159	UDP	56	54657 → 21116 Len=14
251	134.003672	192.168.0.101	155.138.247.159	UDP	56	54657 → 21116 Len=14
252	135.993832	192.168.0.101	155.138.247.159	UDP	56	54657 → 21116 Len=14
253	137.000197	192.168.0.101	155.138.247.159	UDP	56	54657 → 21116 Len=14
254	137.215872	155.138.247.159	192.168.0.101	UDP	60	21116 → 54657 Len=4
255	137.216061	192.168.0.101	155.138.247.159	UDP	126	54657 → 21116 Len=84
259	143.711427	192.168.0.100	224.0.0.251	MDNS	103	Standard query 0x0012 PTR _233637DE._sub._googlecast._tcp.local, "QM" question PTR _googlecast._tcp.local, "QM" question
260	149.594726	192.168.0.101	192.168.0.1	DNS	87	Standard query 0xc4e7 A images-ext-1.discordapp.net
261	149.596317	192.168.0.1	192.168.0.101	DNS	167	Standard query response 0xc4e7 A images-ext-1.discordapp.net A 162.159.133.232 A 162.159.129.232 A 162.159.128.232 A 162.159.134.232 A 162.159.130.232
262	150.003489	192.168.0.101	155.138.247.159	UDP	56	54657 → 21116 Len=14
266	151.997678	192.168.0.101	155.138.247.159	UDP	56	54657 → 21116 Len=14
267	153.995793	192.168.0.101	155.138.247.159	UDP	56	54657 → 21116 Len=14
268	156.003324	192.168.0.101	155.138.247.159	UDP	56	54657 → 21116 Len=14
269	158.002175	192.168.0.101	155.138.247.159	UDP	56	54657 → 21116 Len=14
270	159.999842	192.168.0.101	155.138.247.159	UDP	56	54657 → 21116 Len=14
271	160.788865	192.168.0.1	192.168.0.255	UDP	370	24941 → 20002 Len=328
272	161.001073	192.168.0.101	155.138.247.159	UDP	56	54657 → 21116 Len=14
276	162.003714	192.168.0.101	155.138.247.159	UDP	56	54657 → 21116 Len=14
277	162.004319	192.168.0.101	192.168.0.1	DNS	78	Standard query 0x34e5 A rs-ny.rustdesk.com
278	162.005570	192.168.0.1	192.168.0.101	DNS	94	Standard query response 0x34e5 A rs-ny.rustdesk.com A 155.138.247.159
279	162.216838	155.138.247.159	192.168.0.101	UDP	60	21116 → 54657 Len=4
